sphinx/ext/autodoc/__init__.py

```diff
@@ -983,8 +983,10 @@ def format_args(self, **kwargs: Any) -> str:
                     not inspect.isbuiltin(self.object) and
                     not inspect.isclass(self.object) and
                     hasattr(self.object, '__call__')):
+                self.env.app.emit('autodoc-before-format-args', self.object.__call__, False)
```

Comment:
I don't think this is good way to do this. But we have to let handlers know the target object is bound method or not...